bundle/manifests/dex-operator.clusterserviceversion.yaml

```diff
@@ -350,7 +350,7 @@ spec:
                 env:
                 - name: RELATED_IMAGE_DEX
                   value: quay.io/dexidp/dex:v2.28.1
-                image: quay.io/chrisahl/dex-operator:conditions
+                image: quay.io/chrisahl/dex-operator:infra-tolerations
```

Comment:
I guess here the image must refer to the identitatem one

Reply:
This CSV is just for testing from the dex-operator repo and `make bits`.  Don't think it needs to point to identitatem latest